src/ng/components/view/view.html

```diff
@@ -1,25 +1,32 @@
-<q-grid-core:toolbar position="right">
+<q-grid-core:toolbar ng-if="$view.visibility.toolbar.top"
+                     position="top">
 </q-grid-core:toolbar>
-<q-grid-core:toolbar position="top">
-</q-grid-core:toolbar>
-<div q-grid-core:viewport
-     class="viewport">
-	<table q-grid-core:table
-	       q-grid-core:selection
-	       class="mode-{{$view.mode}}">
-		<thead ng-if="$view.visibility.head"
-		       q-grid-core:head
-		       q-grid-core:sticky
-		       ng-include="$view.templateUrl('head')">
-		</thead>
-		<tfoot q-grid-core:foot
-		       q-grid-core:sticky
-		       ng-include="$view.templateUrl('foot')">
-		</tfoot>
-		<tbody q-grid-core:body
-		       ng-include="$view.templateUrl('body')">
-		</tbody>
-	</table>
+<div class="flexible">
```

Comment:
still not good name, it shows how it layouts internally and not what is that for...btw. wrapper was beetr :D
consider maybe dock-center, if so wrap top, bottom toolbars to div with classes dock-top and dock-bottom 
dock, and right left to dock-left and dock-right